content/redirect.js

```diff
@@ -49,24 +49,34 @@ function resolveDocumentPath(url) {
 }
 
 // Throw if this can't be a redirect from-URL.
-function validateFromURL(url) {
+function validateFromURL(url, checkResolve = true) {
+  // Let's keep vanity urls to /en-US/
+  if (url === "/en-US/") {
```

Comment:
Shouldn't this be something like
```javascript
if (VALID_LOCALES.map(locale => `/${locale}/`).includes(url)) {
```
Just in case the Japanese wants a particular page to simply go back to the home page?

Reply:
Yes, I'm changing that. In all `_redirects.txt` we only had this for _en-US_